scripts/sanitycheck

```diff
@@ -1883,7 +1887,7 @@ class TestInstance:
         return build_only
 
     def create_overlay(self, platform):
-        file = os.path.join(self.outdir, "overlay.conf")
+        file = os.path.join(self.outdir, "testcase_configs.txt")
```

Comment:
It's easier to understand what is going on when files have the correct suffix.

How about

outdir/testcase_configs/overlay.conf

?

Reply:
I think it's a bit "overkill" but I'll give it a try. "Overkill" because there's already "configs" just before .txt and because this filename is an internal implementation detail that no one will ever look at unless something goes wrong.